planner/core/stats.go

```diff
@@ -262,6 +278,9 @@ func (ds *DataSource) accessPathsForConds(conditions []expression.Expression, us
 			}
 		} else {
 			path.index = ds.possibleAccessPaths[i].index
+			if !ds.isInIndexMergeHints(path.index.Name.L) {
```

Comment:
If the hint is like `use_index_merge(t, tb, tc, td)`, we would generate index merge paths like `{tb, tc}`, which does not strictly match the hint.

Reply:
so must generate a index scan on `td`?